haystack/nodes/retriever/dense.py

```diff
@@ -1125,8 +1125,6 @@ def _get_predictions(self, dicts: List[Dict[str, Any]]) -> Dict[str, np.ndarray]
             disable=disable_tqdm,
         ) as progress_bar:
             for batch in data_loader:
-                batch = {key: batch[key].to(self.devices[0]) for key in batch}
```

Comment:
We need to keep this line in order to use the GPU when doing inference.

Reply:
Thanks for catching that, sorry @bogdankostic 🤦‍♂️!